.github/workflows/first-interaction.yml

```diff
@@ -1,10 +1,12 @@
 name: review
-on:
-  push:
-    branches:
-      - main
-  pull_request:
-    branches: [ main ]
+#on:
```

Comment:
Did you disable it on purpose or did you forgot to uncomment?

Reply:
The `first-interaction` actions was experiencing some issues which can be seen hear. Looks like they have close and resolved the issue. Will uncomment and try it again.

https://github.com/actions/first-interaction/issues/101

Reply:
I guess you should bump it to `v1.1.1`

Reply:
We should be able to use `v1` as it will include all updates. 
https://github.com/marketplace/actions/first-interaction